pkg/pool-weighted/contracts/managed/ManagedPoolSettings.sol

```diff
@@ -376,6 +379,22 @@ abstract contract ManagedPoolSettings is NewBasePool, ProtocolFeeCache, IManaged
         emit GradualWeightUpdateScheduled(startTime, endTime, startWeights, endWeights);
     }
 
+    // Join / Exit Enabled
+
+    function getJoinExitEnabled() external view override returns (bool) {
+        return ManagedPoolStorageLib.getJoinsExitsEnabled(_poolState);
+    }
```

Comment:
Small note about naming: I've followed the same style used for the swap flag, which uses singular in settings, but plural in the storage lib.

Maybe consider using the same style for everything?

Reply:
Yeah, I've noticed this as well (and am bugged by it). I'd prefer plural on both but sadly that would muck with the interface so we should standardise on the singular.

Reply:
+1.

Opened https://github.com/balancer-labs/balancer-v2-monorepo/issues/2071, I'll address this in a separate PR so as not to mix things.